cmd/acra_configui/acra_configui.go

```diff
@@ -105,12 +116,37 @@ func SubmitSettings(w http.ResponseWriter, r *http.Request) {
 	w.Write(jsonToServer)
 }
 
+func parseTemplate() {
+	tplPath := fmt.Sprintf(filepath.Join(*staticPath, "index.html"))
+	tplPath, err = utils.AbsPath(tplPath)
+	if err != nil {
+		log.WithError(err).Errorf("no config file[%v]", tplPath)
+	}
+
+	parsedTemplate, err = template.ParseFiles(tplPath)
+	if err != nil {
+		log.WithError(err).Errorf("err while parsing template - %v", tplPath)
+		syscall.Exit(1)
+	}
+}
+
+func parseConfig() []byte {
+	configPath, err_ := utils.AbsPath(fmt.Sprintf("./%s", CONFIG_PATH))
```

Comment:
1. you can read file with relative path without converting to absolute. `CONFIG_PATH` constains relative path `config/blabla` and it can be read without transformations
2. what if user want to use config with custom path like `/var/acra/some.conf`? or how we can change path if we will distribute our package with binary and place config in different place? for example binary will be placed to /usr/local/bin and configs to `/etc/acra/*` or `/usr/local/etc` ?

Reply:
As discussed - will move configs/acraserver_config_vars.yaml to cmd/acra_configui/acraserver_config_vars.go as it's internal thing.
TODO: make run-time config

Reply:
done